thunder/__init__.py

```diff
@@ -471,25 +488,41 @@ def get_computation_and_inputs(*args, **kwargs):
                 epilogue_trc = jit_results.epilogue_trace
                 last_interpreter_log = jit_results.interpreter_log
 
+            prologue_traces = [prologue_trc]
+            computation_traces = [computation_trc]
+
             if epilogue_trc is not None:
                 epilogue_traces = [epilogue_trc]
-                epilogue = epilogue_trc.python_callable()
             else:
                 epilogue_traces = None
-                epilogue = None
 
             cs.last_trace_tracing_stop = time.time_ns()
 
             # Makes the prologue callable
             cs.last_prologue_transformation_start = time.time_ns()
-            protraces = transform_for_execution(
+
+            for transform in early_transforms:
+                prologue_trc, computation_trc, epilogue_trc = transform(
```

Comment:
Q: would we have a spec of `transform: Callable`?

Reply:
I meant to ask whether or not we have a specification of a transform (and optionally a check)